LICENSE

```diff
@@ -27,7 +27,20 @@ The Chapel implementation is composed of two categories of code:
      vim              vim-based syntax coloring                    VIM
 
    third-party/
-     chpldoc-venv     python packages supporting chpldoc           see third-party/README
+     chpldoc-venv/    python packages supporting chpldoc
```

Comment:
Should the second change occur before we release?  It seems unfortunate to go out with a directory name we'd like to change, though I'd be a little worried about breaking all the makefiles after code freeze.